bower.json

```diff
@@ -10,6 +10,7 @@
     "easteregg": "https://github.com/mozilla/fxa-easter-egg.git#724d487c5ed027dfeab789af6dd9308fd88b9961",
     "fxa-content-server-l10n": "https://github.com/mozilla/fxa-content-server-l10n.git",
     "fxa-js-client": "https://github.com/mozilla/fxa-js-client.git#0.1.30",
+    "fxa-password-strength-checker": "https://github.com/mozilla/fxa-password-strength-checker.git#master",
```

Comment:
Pin this to a known good sha, importing `#master` is considered bad practice because someone could merge a breaking change to master that causes us to break.